### PR TITLE
Enforced Airbnb style guide with Eslint

### PR DESCRIPTION
**Pull request by 0xPsyop**
Added scripts  "lint" and "lint-fix" for easy use

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (pavelkeyzik) [inline .eslintrc.json]: Can we fix tab spaces here? We use 2, and in my case when I save this file, prettier will update format

File: src/core/messages.js [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+  'This is an excellent improvement',
+  'Nice and tidy, great job!',
+  'Outstanding job!',
+  'Nicely done!',
+  'Way to go!',
+  "Good feature, let's merge it!",
 ];
 
-export { messages };
+export default messages;
```
Comment (pavelkeyzik) [inline src/core/messages.js]: Can we ignore this rule? From my experience default export always makes issues with the naming when someone use it like `import messagesList from 'messages';`. Sometimes it's harder to keep in mind that messagesList it's just messages and it's harder to refactor that

**Review by pavelkeyzik — CHANGES_REQUESTED** [review]
Hey @Iam-SilentEyes 👋 Thanks for working on this feature. I'm so happy to see this PR. May I ask you to fix some little issues from comments and one more thing.

We're using Prettier with default configuration and when Prettier formats file, it ignores Airbnb configuration. That way we get a problems like when we save file, Prettier converts `''` to `""` and ESLint is angry 😆

File: .vscode/extensions.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "recommendations": [
+        "mflo999.lintel",
```
Comment (pavelkeyzik) [inline .vscode/extensions.json]: Can you remove that from recommendations? As the configuration helper is not something everyone should have while working on the project, so I guess it's not something that should live in a recommendations section 😊

**Comment by pavelkeyzik** [comment]
Hey @Iam-SilentEyes 👋  I saw you commited fixes, but there is still some errors. Just want to know if you're still workgin on that 😊

File: .vscode/extensions.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (pavelkeyzik) [inline .vscode/extensions.json]: Can we remove this file, as we don't have any recommendations here?

File: .eslintrc.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
+    {
```
Comment (pavelkeyzik) [inline .eslintrc.json]: Tab size is 4 spaces here, instead of 2. Can you fix that?

File: src/core/messages.js [hunk 6]
```diff
@@ -1,27 +1,27 @@
 const messages = [
-  "Nice job",
-  "Looks good",
-  "Clean fix!",
-  "Well done",
-  "Thank you! Really nice PR",
-  "Great job",
-  "Looks amazing",
+  'Nice job',
```
Comment (pavelkeyzik) [inline src/core/messages.js:2]: ESLint changes `""` to `''`, but Prettier changes to `''`. Prettier runs after ESLint, so the problem will be that when we save file, our quotes will be `''` and the ESLint will have errors

**Review by pavelkeyzik — CHANGES_REQUESTED** [review]

**Comment by Iam-SilentEyes** [comment]
Hey @pavelkeyzik is anything wrong again ? If not maybe you can accept it :)

**Comment by pavelkeyzik** [comment]
As you may see, our PR contains new ESLint configuration, but at the same time we have changes in our files. We don't want it, as we should configure ESLint to follow project styles. Also, ESLint should work with Prettier. For example, you can see that emojis file is updated, because ESLint changes "" to '', but we use Prettier and it will update '' to "" after ESLint updates. It means that it's impossible to save file with new ESLint rule as it's conflicts with Prettier. Can you try to install VSCode extension and configure it to format code on save file? That way you should see a problem

**Pull request closed without merging**